vega_sim/api/governance.py

```diff
@@ -421,11 +421,19 @@ def _make_and_wait_for_proposal(
     )
     logger.debug("Waiting for proposal acceptance")
 
-    time_forward_fn()
-    proposal = wait_for_acceptance(
-        proposal.reference,
-        lambda p: _proposal_loader(p, data_client),
-    )
+    # Allow one failure, forward once more
+    try:
+        time_forward_fn()
+        proposal = wait_for_acceptance(
+            proposal.reference,
+            lambda p: _proposal_loader(p, data_client),
+        )
+    except ProposalNotAcceptedError:
```

Comment:
When I was a lad I was told in school that using exceptions for "normal flow control" is bad practice... but I don't see why it would really be a problem here - we never expect this to be "fast".